src/crimson/os/seastore/segment_manager/zns.cc

```diff
@@ -489,14 +510,12 @@ Segment::close_ertr::future<> ZNSSegmentManager::segment_close(
   LOG_PREFIX(ZNSSegmentManager::close);
   return seastar::do_with(
     blk_zone_range{},
-    [=] (auto &range){
+    [=] (auto &range) {
       range = make_range(
-	id, 
-	metadata.zone_size, 
-	metadata.block_size, 
-	metadata.first_segment_offset);
+	id,
+	metadata.zone_size_sectors);
```

Comment:
This is `segment_size_sects`, should be calculated from `segment_size`, not `zone_size`.

Reply:
Done